src/de/opendiabetes/vault/data/VaultDao.java

```diff
@@ -117,6 +117,9 @@ public static void finalizeDb() throws IOException {
      */
     public static void initializeDb() throws SQLException {
         //TODO combine logging
+        // Der OR Mapper benutzt (soweit ich mich erinnere) Log4J zum loggen und ich möchte gerne den Logger
```

Comment:
Either provide comment in english, fix or leave it out completely and specify as task